components/local_date_time/local_date_time.tsx

```diff
@@ -29,37 +28,44 @@ type Props = {
 }
 
 export default class LocalDateTime extends React.PureComponent<Props> {
-    public render() {
+    getFormattedTime = () => {
         const {
             enableTimezone,
             eventTime,
             timeZone,
             useMilitaryTime,
         } = this.props;
 
-        const date = eventTime ? new Date(eventTime) : new Date();
+        const value = eventTime ? new Date(eventTime) : new Date();
```

Comment:
you can change this to momentDate = eventTime ? moment(eventTime) : moment();

Reply:
Thanks, that's neat.